AsyncDisplayKit/ASEditableTextNode.mm

```diff
@@ -189,6 +191,25 @@ - (void)setOpaque:(BOOL)opaque
 @synthesize delegate = _delegate;
 
 #pragma mark -
+@dynamic textView;
+
+- (UITextView *)textView
+{
+  ASDisplayNodeAssertMainThread();
+  if (!_textView) {
+    _textView = [[_ASDisabledPanUITextView alloc] initWithFrame:CGRectZero textContainer:_textKitComponents.textContainer];
+  }
+  return _textView;
+}
+
+- (void)setTextView:(UITextView *)textView
```

Comment:
Seems like we should get rid of this since presumably the _ASDisabledPanUITextView is important.